composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "fba4e7b84789ee84bfa3a46b5bba64c8",
```

Comment:
Is this change necessary? It's not in your last commit so may break the tests

Reply:
I didn't commit the lock file. How do I fix this?